plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TableStatisticsReader.java

```diff
@@ -140,6 +151,8 @@ public static TableStatistics makeTableStatistics(
                 .map(column -> Maps.immutableEntry(column.fieldId(), column.type()))
                 .collect(toUnmodifiableMap(Map.Entry::getKey, Map.Entry::getValue));
 
+        Domain pathDomain = getPathDomain(effectivePredicate);
+        Domain fileModifiedTimeDomain = getFileModifiedTimePathDomain(effectivePredicate);
         TableScan tableScan = icebergTable.newScan()
                 // Table enforced constraint may include eg $path column predicate which is not handled by Iceberg library TODO apply $path and $file_modified_time filters here
```

Comment:
This TODO can be removed now right?

cc @findepi

Reply:
Removed